benchmark/queries/ldbc-sf100/shortest_path_ldbc100/q37.benchmark

```diff
@@ -0,0 +1,5 @@
+-NAME q37
```

Comment:
Open a saperate PR to merge these benchmarks into master

Reply:
Raised a separate PR